calico/networking/configuring/bgp-to-workload.mdx

```diff
@@ -0,0 +1,302 @@
+---
+description: Configure BGP peering with nested clusters running on KubeVirt VMs
+---
+
+# Configure BGP peering with nested clusters running on KubeVirt VMs
+
+## Big picture
+
+Configure BGP peering with $[prodname] nodes running on KubeVirt VMs that are
+running on your Kubernetes cluster.
+
+## Value
+
+Run nested Kubernetes clusters in VMs on your parent Kubernetes cluster.
+Avoid network overlay overheads; extend layer 3 network reachability all the
+way from pods running in the nested cluster to your physical network
+infrastructure.
+
+## Concepts
+
+This guide assumes you have a good understanding of BGP and $[prodname]'s BGP
+peering options.
+
+The guide makes use of several $[prodname] resources, including:
+
+- [BGPPeer](../../reference/resources/bgppeer.mdx) - Used to configure peerings
+  both with external BGP routers and with BGP-enabled workloads.
+- [BGPFilter](../../reference/resources/bgpfilter.mdx) - Defines a BGP filter
+  to control which routes are imported or exported between BGP peers.
+- [BGPConfiguration](../../reference/resources/bgpconfig.mdx) - Configures global
+  BGP settings for the cluster.
+
+The [BGP Guide](./bgp.mdx) provides a general overview of these concepts.
+
+## Supported BGP topologies
+
+Both parent and nested clusters must be configured to use non-overlay BGP
+networking.
+
+eBGP must be used both for peering between parent cluster and top-of-rack
+router and for peering between parent cluster and nested workload cluster.
+Each of the ToR, parent cluster, and nested cluster must have a unique AS number.
+
+iBGP peerings to workloads or ToR routers are not supported:
+
+- iBGP to the ToR router is not supported because there is currently no way to
+  configure "next hop self" behaviour on that peering (so the ToR would receive
+  the nested cluster's routes with the workload's IP as next hop, rather than
+  the parent cluster node's IP).
+
+- iBGP to the nested cluster has not been tested. It may work with appropriate
```

Comment:
Is it worth mentioning this and risk giving the reader false hope in the case where it doesnt work?

Reply:
I'm not sure; I wanted to say something more than "iBGP is not supported" because users will try it and, if it seems to work, then we're in the grey area anyway.